lib/jsdoc/util/logger.js

```diff
@@ -130,34 +141,26 @@ function addPrefix(args, prefix) {
     return updatedArgs || args;
 }
 
-// TODO: document events
-function wrapLogFunction(name, func) {
-    var eventName = 'logger:' + name;
-    var upperCaseName = name.toUpperCase();
-    var level = LEVELS[upperCaseName];
-    var prefix = PREFIXES[upperCaseName];
-
-    return function() {
-        var loggerArgs;
 
-        var args = Array.prototype.slice.call(arguments, 0);
-
-        if (logLevel >= level) {
-            loggerArgs = addPrefix(args, prefix);
-            func.apply(null, loggerArgs);
-        }
+var Logger = function () {
+    this.handlers = {};
```

Comment:
Are you suggesting something like that:

```
logger.on('logger:warn', function () {
   var logArgs = _.toArray(arguments);
   var level = LEVELS.WARN;
  if (typeof console === 'object') {
    if (this._canBeLogged(level)) {
      console.log.apply(this, addPrefix(logArgs, level));
    }
});
// .. same thing for other levels..
```

But how will people override it? Events will just pile up. I thought the goal was to let users be able to customize the logger handlers?